src/Schema/Factories/FieldFactory.php

```diff
@@ -193,9 +193,7 @@ protected function wrapResolverWithValidation(\Closure $resolver, Collection $in
                     ]
                 );
 
-                if ($validator->fails()) {
-                    throw with(new ValidationError('validation'))->setValidator($validator);
-                }
+                $validator->validate();
```

Comment:
Since we're always throwing a `Illuminate\Validation\ValidationException` here is there any way to handle the error output for this? 

For instance, on the frontend of a project we're placing all the validation errors in the `errors.0.validation` array along w/ setting `errors.0.status_code` = 422. Am I able create my own Error Handler class/callback to be used in place of the default Handler to handle custom output (i.e., putting all validation errors inside the first error returned from the server)?

If we don't currently have a way of doing that in this PR it seems that we could do the following:

```php
class Handler
{
    protected $handler = null;

    public function setHandler(\Closure $handler) 
    {
        $this->handler = $handler;
    }

    /**
     * @param array|GraphError[] $errors
     * @return array
     */
    public function handler(array $errors) : array
    {
        if ($this->handler) {
            return call_user_func($this->handler, $errors);
        }
        
        // ...
    }
}
```

and make the Handler a singleton so in a service provider we could do the following:

```php
app(\Nuwave\Lighthouse\Support\Exceptions\Handler::class)
    ->setHandler(function (array $errors) {
        // custom error handler logic...
    });

// or, instead of creating a singleton we could do this:

app('graphql')->exceptionHandler()->setResolver(function (array $errors) {
    // ...
});
```

If I completely spaced out and missed this, my apologizes! Looking at the `itCanThrowValidationExceptions` it just looks like multiple validation errors will put multiple errors in the response and since `ValidationException` are always handled the same way we might not have an option to override the output.

Great work on this PR @olivernybroe!!!

Reply:
The handler is resolved with Laravel, so you can just make a binding for it. However we can make an interface and use that if that is more intuitive when making a binding.

To get the old way of showing exceptions, just extend the handler and bind your extended class in the container. Then override render method and call parent render at the end. Here you get the exception, so just make an if for ValidationException and specify how you want it rendered.

Reply:
Awesome, let's go the interface route 👍